src/utils/get-code-from-ast.ts

```diff
@@ -10,7 +10,11 @@ import { newLineCharacters } from '../constants';
  * @param nodes all imports
  * @param originalCode
  */
-export const getCodeFromAst = (nodes: Statement[], originalCode: string) => {
+export const getCodeFromAst = (
+    nodes: Statement[],
+    originalCode: string,
+    interpreter: InterpreterDirective | null | undefined,
```

Comment:
```suggestion
    interpreter?: InterpreterDirective
```
May be undefined or ImperativeDirective